core/src/main/scala/org/apache/spark/sql/hive/HiveClientUtil.scala

```diff
@@ -272,8 +260,10 @@ class HiveClientUtil(val sparkContext: SparkContext) extends Logging {
     val sc = sparkContext
     Property.MetaStoreDBURL.getOption(sparkConf) match {
       case Some(url) =>
-        val driver = Property.MetaStoreDriver.getOption(sparkConf).orNull
-        (false, url, driver)
+//        val driver = Property.MetaStoreDriver.getOption(sparkConf).orNull
+//        (false, url, driver)
+        throw new IllegalArgumentException(s"Invalid property ${Property.MetaStoreDBURL}. " +
+            s"SnappyData does not allow use of property ${Property.MetaStoreDBURL}")
```

Comment:
Can we remove these properties from Literals.scala. In fact there is no need to read these properties and resolveMetaStoreDBProps can return (String, String) instead of (Boolean, String, String)